src/FsAutoComplete/CodeFixes.fs

```diff
@@ -37,7 +38,7 @@ module Types =
       SourceDiagnostic: Diagnostic option
       Kind: FixKind }
 
-  type CodeFix = CodeActionParams -> Async<Fix list>
+  type CodeFix = CodeActionParams -> Async<Result<Fix list, string>>
```

Comment:
making this AsyncResult makes authoring codefixes nicer, IMO. we do the unification and logging in the LSP endpoint itself now.